requirements-ext.txt

```diff
@@ -1,7 +1,7 @@
 cached_property
 cachetools
+ipympl
```

Comment:
Should this really be installed by default? Seems like it might break some of this separation that we are trying to introduce.

Reply:
Good spot, this is required for tests, but shouldn't be there by default!